scripts/netlify-headers.js

```diff
@@ -0,0 +1,99 @@
+var AssetGraph = require('assetgraph');
```

Comment:
BTW, you can use ES6 in `scripts/*.js`, or at least as far as Node.js v4.x takes it.